appeals/e2e/cypress/e2e/back-office-appeals/uploadDocsAppellantCase.spec.js

```diff
@@ -0,0 +1,33 @@
+// @ts-nocheck
+/// <reference types="cypress"/>
+
+import { users } from '../../fixtures/users';
+import { CaseDetailsPage } from '../../page_objects/caseDetailsPage.js';
+import { ListCasesPage } from '../../page_objects/listCasesPage';
+import { DateTimeSection } from '../../page_objects/dateTimeSection';
+import { urlPaths } from '../../support/urlPaths.js';
+import { tag } from '../../support/tag';
+import { happyPathHelper } from '../../support/happyPathHelper.js';
+
+const listCasesPage = new ListCasesPage();
+const dateTimeSection = new DateTimeSection();
+const caseDetailsPage = new CaseDetailsPage();
+
+describe('Upload Documents to appellant case', () => {
+	beforeEach(() => {
+		cy.login(users.appeals.caseAdmin);
+	});
+
+	it('Upload documents to appellent case', { tags: tag.smoke }, () => {
+		cy.createCase().then((caseRef) => {
+			happyPathHelper.assignCaseOfficer(caseRef);
+			caseDetailsPage.clickReviewAppellantCase();
+			caseDetailsPage.clickAddAgreementToChangeDescriptionEvidence();
+			caseDetailsPage.uploadSamplePdf();
+			caseDetailsPage.clickButtonByText('Continue');
+			caseDetailsPage.clickButtonByText('Confirm');
+			caseDetailsPage.clickButtonByText('Confirm');
+			caseDetailsPage.verifyAnswerSummaryValue('sample-doc.pdf');
```

Comment:
A refactor for the future, it would be nice to have the file names stored somewhere else the same way we use the `urlPaths.js` file to hold a list of URLs 

In programming you generally don't want to have plain strings in your code, as they're ripe for misspelling and make it harder to refactor. So we could apply the same principle to the `Continue` and `Confirm` reference. However I wouldn't rush to do this, as could easily become premature optimization

Reply:
Thank you, completely agree,  definitely up for discussing this in our next catch-up